core/trashcan.js

```diff
@@ -372,6 +372,34 @@ Blockly.Trashcan.prototype.contentsIsOpen = function() {
   return this.flyout.isVisible();
 };
 
+/**
+ * Opens the trashcan flyout.
+ */
+Blockly.Trashcan.prototype.openContents = function() {
```

Comment:
I have a slight preference for `openFlyout` and `closeFlyout` instead of `openContents` and `closeContents`

Reply:
sgtm